forward/ip_frag_reassembly.lua

```diff
@@ -0,0 +1,68 @@
+local ffi = require("ffi")
+
+local IPFragReassembly = {}
+
+function IPFragReassembly:new()
+   return setmetatable({
+      frag_pkts = {}
+   }, {
+      __index = IPFragReassembly
+   })
+end
+
+-- Returns a binary containing the reassembled packet data
+function IPFragReassembly:process_packet(five_tuple_str, offset, mf, payload, payload_len)
```

Comment:
Could you explain what this function does a little more?  I don't think I quite understand the comment.